talk/basicconcepts/scopesnamespaces.tex

```diff
@@ -183,4 +183,31 @@
     \end{cppcode*}
   \end{alertblock}
 \end{frame}
+
+\begin{frame}[fragile]
+  \frametitlecpp[98]{Using namespace std}
+  \begin{block}{Purpose}
+    \begin{itemize}
+      \item Directive to make all members of the \texttt{std} namespace visible
+      \item Can be used for convenience, but risk of name clashes
+      \item OK in implementation files or scopes
+      \item Alternative: using declaration (e.g.\ \cppinline{using std::cout;})
```

Comment:
Would you say such a using declaration is fine in a public header? I don't think so, but it is for sure less dangerous.

I am afraid that the slide could be interpreted as: Use using declarations instead in a header file!

Reply:
That's not what I meant, indeed. Needs rephrasing.

Reply:
Rephrasing done.